Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -264,9 +268,19 @@ open class AVFoundationPlayback: Playback {
     }
 
     private func seekToStartAtIfNeeded() {
-        if canStartAt {
-            seek(startAt)
-        }
+        guard canStartAt else { return }
+        
+        seek(startAt)
+    }
+    
+    private func seekToLiveStartTimeIfNeeded() {
+        guard canLiveStartAt, let liveStartTime = liveStartTime else { return }
+        
+        let positionToSeek = liveStartTime - epochDvrWindowStart
+        
+        if positionToSeek < 0 || positionToSeek > duration { return }
```

Comment:
I think this rule can be encapsulated in a method or var to be more expressive. What do you think?

Reply:
I have the same opinion

Reply:
Done